app/waf/src/controllers/endpointpolicy.controller.ts

```diff
@@ -119,28 +99,110 @@ export class EndpointpolicyController {
     await this.endpointpolicyRepository.updateById(id, endpointpolicy);
   }
 
-  @put(prefix + '/endpointpolicies/{id}', {
+  @del(prefix + '/endpointpolicies/{id}', {
     responses: {
       '204': {
-        description: 'Endpointpolicy PUT success',
+        description: 'Endpointpolicy DELETE success',
       },
     },
   })
-  async replaceById(
-    @param.path.string('id') id: string,
-    @requestBody() endpointpolicy: Partial<Endpointpolicy>,
-  ): Promise<void> {
-    await this.endpointpolicyRepository.replaceById(id, endpointpolicy);
+  async deleteById(@param.path.string('id') id: string): Promise<void> {
+    await this.endpointpolicyRepository.deleteById(id);
   }
 
-  @del(prefix + '/endpointpolicies/{id}', {
+  @post(prefix + '/endpointpolicies/{endpointpolicy_id}/rules', {
```

Comment:
endpointpolicy_id --> endpointpolicyId

This parameter name will be exposed via API spec, so we need to make its naming be consistent with others.